README.md

```diff
@@ -175,6 +175,14 @@ code.
 ### Android Quirks
 
 - The `duration` parameter is not supported.  Recording lengths can't be limited programmatically.
+- Android uses intents to launch the camera activity on the device to capture
+images, and on phones with low memory, the Cordova activity may be killed.  In this
+scenario, the result from the plugin call will be delivered via the resume event.
+See [the Android Lifecycle guide][android_lifecycle]
```

Comment:
This is a reference style markdown link and I think you missed copying over the actual link definition at the bottom of the README. Should point to http://cordova.apache.org/docs/en/latest/guide/platforms/android/index.html#lifecycle-guide